learn_ai/assignments/text_classification_assignment_1.ipynb

```diff
@@ -0,0 +1,1808 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        qdrant_client.upload_points(collection_name='text-classification', points=[point])
</pre><p>We're inserting one point at a time, inside the loop here! I believe this line should be outside the loop?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/FuturePathAI/Learn-AI-Engineering/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>